leverage/path.py

```
@@ -3,6 +3,7 @@
 """
 
 import os
+import pathlib
```

Ruff (0.8.2)

`pathlib` imported but unused

Remove unused import: `pathlib`

(F401)